pdr_backend/sim/sim_engine.py

```diff
@@ -57,7 +57,9 @@ def __init__(
         self.exchange = exchange_mgr.exchange(
             "mock" if mock else ppss.predictoor_ss.exchange_str,
         )
-
+        self.position_open = ""  # long, short, ""
+        self.position_size = 0  # amount of tokens in position
+        self.position_worth = 0  # amount of USD in position
```

Comment:
Instead of "position_open" and "position_size", why not just have `position_size` and allow it to be negative? Maybe better, up to you.